README.md

```diff
@@ -117,12 +121,12 @@ spec:
         speed: 1000
       ipv4:
         address:
-        - ip: 10.0.0.2
+        - ip: 192.0.2.2
           prefix-length: 24
         dhcp: true
         enabled: true
         neighbors:
```

Comment:
Not supported in *nmstate* so it is futuristic.

Reply:
Static addresses are not supported?

Reply:
All those fields are included in kubernetes-nmstate State type. It is possible that some of them are broken. However, testing will be added in following PR and cover all examples.

Reply:
Done